Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -829,7 +829,10 @@ namespace AssetProcessor
                 // The product file path is always lower cased, we can't check that for existance.
```

Comment:
trivial, and not code you've touched, but while you're here it might be worth fixing the typo: existance -> existence